pkg/webrtc_ext/config.go

```diff
@@ -0,0 +1,9 @@
+package webrtc_ext
+
+// Configuration of the WebRTC API for the SFU.
+type Config struct {
+	// Enable simulcast extension.
+	EnableSimulcast bool `yaml:"simulcast"`
+	// Pulibc IP address of the SFU.
+	PublicIP string `yaml:"ip"`
```

Comment:
As @fkwp suggested, it would be great if this could be an array

Reply:
Agree, makes sense!